databases.go

```diff
@@ -70,6 +71,7 @@ type Database struct {
 	Cluster                    bool              `json:"cluster"`
 	PeriodicBackupsEnabled     bool              `json:"periodic_backups_enabled"`
 	PeriodicBackupsScheduledAt []int             `json:"periodic_backups_scheduled_at"` // Hours of the day of the periodic backups (UTC)
+	MaintenanceWindow          MaintenanceWindow `json:"maintenance_window"`
```

Comment:
@SCedricThomas The property `maintenance_window` is not mentionned in the documentation : https://developers.scalingo.com/databases/databases
Is it normal ?

Reply:
This feature is still in work in progress and not used on production at the moment but you are right! I will make an update on the documentation as soon as possible